src/main/java/org/folio/rest/impl/OrdersImpl.java

```diff
@@ -143,19 +143,18 @@ public void putOrdersById(String orderId, String lang, CompositePurchaseOrder co
 
   @Override
   @Validate
-  public void postOrdersLinesById(String orderId, String lang, PoLine poLine, Map<String, String> okapiHeaders,
-                                  Handler<AsyncResult<javax.ws.rs.core.Response>> asyncResultHandler, Context vertxContext) {
+  public void postOrdersOrderLines(String lang, PoLine poLine, Map<String, String> okapiHeaders,
+                                   Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
     final HttpClientInterface httpClient = AbstractHelper.getHttpClient(okapiHeaders);
     PostOrderLineHelper helper = new PostOrderLineHelper(httpClient, okapiHeaders, asyncResultHandler, vertxContext, lang);
 
     logger.info("Creating POLine to an existing order...");
-
-    if (poLine.getPurchaseOrderId() == null) {
-      logger.info("POLine without id. Set id from path url: " + orderId);
-      poLine.setPurchaseOrderId(orderId);
-    }
-    loadConfiguration(okapiHeaders, vertxContext, logger).thenAccept(config -> {
-      if (orderId.equals(poLine.getPurchaseOrderId())) {
+    loadConfiguration(okapiHeaders, vertxContext, logger)
+      .thenAccept(config -> {
+        String orderId = poLine.getPurchaseOrderId();
+        if (orderId == null) {
+          throw new ValidationException(MISMATCH_BETWEEN_ID_IN_PATH_AND_PO_LINE, "id_not_exists");
```

Comment:
Please use instead of `MISMATCH_BETWEEN_ID_IN_PATH_AND_PO_LINE` something like `MISSING_ORDER_ID`

Reply:
constant definition updated